config/default/layout_builder_styles.style.section_alignment_start.yml

```diff
@@ -0,0 +1,10 @@
+uuid: 03cf5420-de43-4f89-9f93-8f37e819c0ae
+langcode: en
+status: true
+dependencies: {  }
+id: section_alignment_start
+label: 'Alignment: Start'
```

Comment:
Should this be 'Vertical alignment:'?

Reply:
I think that's a good suggestion since there could eventually be Horizontal alignment.   @joewhitsitt What's your take on the horizontal and vertical alignment options that you built into the paragraph sections?

Reply:
Yeah, I would prepare for a horizontal in the future by naming this one Vertical. We prefixed "content" because people were confused when adding this setting on a section what it actually did.

We also used labels that more users identify with (e.g. top, bottom, middle, left, center, right) instead of flex terms. "none" was an after thought to achieve equal-heights.

<img width="444" alt="Screen Shot 2020-07-10 at 8 54 23 AM" src="https://user-images.githubusercontent.com/4663676/87162628-cd5eb480-c28b-11ea-9ed8-343a02a5e85f.png">

Moving the layouts world, all of our stuff was on a row though, not able to stack like you have done here, so I am not sure what would make the most sense to users.

Reply:
@joewhitsitt thanks for chiming in.   I changed the name to "Vertical alignment: Start" for now.